samples/wfi32iot/src/config/pic32mz_w1/tasks.c

```diff
@@ -0,0 +1,388 @@
+    }
+}
+
+
+
+TX_THREAD      _SYS_CMD_Task_TCB;
+uint8_t*       _SYS_CMD_Task_Stk_Ptr;
+
+static void _SYS_CMD_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        SYS_CMD_Tasks();
+        tx_thread_sleep((ULONG)(10 / (TX_TICK_PERIOD_MS)));
+    }
+}
+
+
+TX_THREAD      _APP_IDLE_Task_TCB;
+uint8_t*       _APP_IDLE_Task_Stk_Ptr;
+
+static void _APP_IDLE_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        /* IDLE Task will be executed if there are not tasks in ready state */
+
+    }
+}
+
+TX_THREAD      _APP_Task_TCB;
+uint8_t*       _APP_Task_Stk_Ptr;
+
+TX_THREAD      _APP_PIC32MZ_W1_Task_TCB;
+uint8_t*       _APP_PIC32MZ_W1_Task_Stk_Ptr;
+
+
+static void _APP_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        APP_Tasks();
+        tx_thread_sleep((ULONG)(50 / (TX_TICK_PERIOD_MS)));
+    }
+}
+
+static void _APP_PIC32MZ_W1_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        APP_PIC32MZ_W1_Tasks();
+        tx_thread_sleep((ULONG)(50 / (TX_TICK_PERIOD_MS)));
+    }
+}
+
+
+
+
+TX_THREAD      _SYS_USB_DEVICE_Task_TCB;
+uint8_t*       _SYS_USB_DEVICE_Task_Stk_Ptr;
+
+static void _SYS_USB_DEVICE_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        /* USB Device layer tasks routine */
+        USB_DEVICE_Tasks(sysObj.usbDevObject0);
+        tx_thread_sleep((ULONG)(10 / (TX_TICK_PERIOD_MS)));
+    }
+}
+
+
+
+
+TX_THREAD      _SYS_FS_Task_TCB;
+uint8_t*       _SYS_FS_Task_Stk_Ptr;
+
+static void _SYS_FS_Tasks( ULONG thread_input )
+{
+    while(1)
+    {
+        SYS_FS_Tasks();
+        tx_thread_sleep((ULONG)(10 / (TX_TICK_PERIOD_MS)));
+    }
+}
+
+
+TX_THREAD   _WDRV_PIC32MZW_Task_TCB;
+uint8_t*    _WDRV_PIC32MZW_Task_Stk_Ptr;
+
+static void _WDRV_PIC32MZW_Tasks(ULONG thread_input)
+{
+    while(1)
+    {
+        SYS_STATUS status = SYS_STATUS_UNINITIALIZED;
+        WDRV_PIC32MZW_Tasks(sysObj.drvWifiPIC32MZW1);
+        status = WDRV_PIC32MZW_Status(sysObj.drvWifiPIC32MZW1);
+        if ((SYS_STATUS_ERROR == status) || (SYS_STATUS_UNINITIALIZED == status))
+        {
+            tx_thread_sleep((ULONG)(50 / (TX_TICK_PERIOD_MS)));
+        }
+    }
+}
+
+void tx_application_define(void* first_unused_memory)
+{
+    (void)first_unused_memory;
+    /* Create a byte memory pool from which to allocate the thread stacks. */
+    tx_byte_pool_create(&byte_pool_0, "byte pool 0", get_byte_pool(), TX_BYTE_POOL_SIZE);
+
+    /* Maintain system services */
+
+#if 0
+    tx_byte_allocate(&byte_pool_0,
+        (VOID **) &_SYS_CMD_Task_Stk_Ptr,
+        SYS_CMD_RTOS_STACK_SIZE,
+        TX_NO_WAIT
+    );
+
+    tx_thread_create(&_SYS_CMD_Task_TCB,
+        "SYS_CMD_TASKS",
+        _SYS_CMD_Tasks,
+        0,
+        _SYS_CMD_Task_Stk_Ptr,
+        SYS_CMD_RTOS_STACK_SIZE,
+        SYS_CMD_RTOS_TASK_PRIORITY,
+        SYS_CMD_RTOS_TASK_PRIORITY,
+        TX_NO_TIME_SLICE,
+        TX_AUTO_START
+    );
+
+    if (status) { while (1); };
+#endif
```

Comment:
Is this code block (inside the #if 0) still useful?

Reply:
With #if 0, I try to indicate and keep track of what I changed/overrode with respect to the baseline of the project that we pick up so that the next time we upmerge to the latest, we can see which things are new more clearly.

Ths CMD_TASK is not useful to us (if that is what you are asking) because we are tight on RAM and I had to eliminate as much as I safely thought I could form the original startup in order to free up some RAM.